File: test/fixtures/python-hello/handler.py
import json


def handler(event):
    return {
        "statusCode": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps({
            "message": "Hello from Python!",
            "method": event["method"],
            "path": event["path"],
        }),
    }

File: test/fixtures/node-hello/handler.js
module.exports.handler = async function (event) {
  return {
    statusCode: 200,
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({
      message: 'Hello from Node.js!',
      method: event.method,
      path: event.path,
    }),
  };
};
